### PR TITLE
separate console and logscribe log levels

### PR DESCRIPTION
**Pull request by Intizar-T**
# Description

Please include a summary of the changes and the related issue.
Please also include relevant motivation and context.
List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: node/pkg/logscribeconsumer/app.go [hunk 1]
```diff
@@ -75,11 +74,16 @@ func (a *App) Write(p []byte) (n int, err error) {
...
-	if !a.isLogLevelValid(res) {
+	// check if the log should be sent to logscribe
+	if isLogLevelValid(res, a.Level) {
+		a.buffer <- res
+	}
+
+	// check if the log should be written to the console
+	if !isLogLevelValid(res, DefaultLogConsoleLevel) {
 		return 0, nil
 	}
```
Comment (nick-bisonai) [inline node/pkg/logscribeconsumer/app.go:85]:
```suggestion
	// check if the log should be written to the console
	if !isLogLevelValid(res, DefaultLogConsoleLevel) {
		return 0, nil
	}
	a.buffer <- res
```
as far as I understand, logs will be written in console through 
if you already called zerolog.SetGlobalLevel(), you won't need to make additional post processing

```
_, err = a.consoleWriter.Write(p)
	if err != nil {
		return 0, err
	}
```

Comment (Intizar-T) [reply node/pkg/logscribeconsumer/app.go:85]: hmm i see... but shouldn't we still check if the log level is above the logscribe/db level before we write the buffer channel?

Comment (nick-bisonai) [reply node/pkg/logscribeconsumer/app.go:85]: awh so you put LogConsoleLevel there, I think you can
```
if !isLogLevelValid(res, logStoreLevel) {
		return 0, nil
	}
	a.buffer <- res
```

Comment (Intizar-T) [reply node/pkg/logscribeconsumer/app.go:85]: got it, will give it a try!

**Comment by nick-bisonai** [comment]
I think if `POST_TO_LOGSCRIBE` is set to false, url connectivity check on initialization should be skipped

**Comment by nick-bisonai** [comment]
please check the test code, let's merge and deploy once it's passed

**Pull request merged**